api/pom.xml

```diff
@@ -103,6 +103,24 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-baseline-maven-plugin</artifactId>
+                <version>4.2.0</version>
+                <configuration>
+                    <base>
+                        <version>2.1.0</version>
```

Comment:
For 2.2 we will need to set the base to 2.0.0, because we are breaking compatibility with 2.1.0 by undoing the incompatible change about gauges.